scripts/confirm_lb_active.sh

```diff
@@ -22,28 +25,69 @@ if [[ -z "${RESOURCE_GROUP_ID}" ]]; then
     exit 1
 fi
 
+if [[ -z "${PRIVATE_ENV}" ]]; then
+    PRIVATE_ENV=false
+fi
+
 # Login to ibmcloud with cli
 attempts=1
-until ibmcloud login -q -r "${REGION}" -g "${RESOURCE_GROUP_ID}" || [ $attempts -ge 3 ]; do
-    attempts=$((attempts + 1))
-    echo "Error logging in to IBM Cloud CLI..." >&2
-    sleep 5
-done
+if [ "$PRIVATE_ENV" ]; then
+    until ibmcloud login -q -r "us-south" -a private.cloud.ibm.com || [ $attempts -ge 3 ]; do
```

Comment:
It should be possible to get the token through  
data "ibm_iam_auth_token" "tokendata" {}

which I think is going through the private IAM token endpoint if the provider is configured so.